image-index.md

```diff
@@ -55,7 +55,10 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the version of the operating system targeted by the referenced blob.
+        Implementations MAY refuse to use manifests where `os.version` is not known to work with the host OS version.
+        When `os` is `windows`, image indexes SHOULD use, and implementations SHOULD understand quad-notation values of `Major.Minor.Build.Revision`. e.g. `10.0.14393.1066`.
```

Comment:
I [still][1] [think][2] we want this to be implementation-defined on Windows to avoid having to specify the [fuzzy matching][3] needed for it to be useful.  And @jstarks seemed to be [ok with that approach][4].

[1]: https://github.com/opencontainers/image-spec/pull/651#discussion_r112993623
[2]: https://github.com/opencontainers/image-spec/pull/632#issuecomment-295021478
[3]: https://github.com/opencontainers/image-spec/pull/632#issuecomment-295018429
[4]: https://github.com/opencontainers/image-spec/pull/632#issuecomment-295498056